### ISSUE TITLE
Bare task names list for completion

### ISSUE THREAD

**Issue by rafi** [tj/robo#46]
It would be nice to have a way to list task names without colors and formatting so it could be consumed by a bash/zsh/fish completion script for the task name. Something like `robo -q` ?

Adding this to `robo.yml`:
```yaml
templates:
  list: |-
    {{range .Tasks}}{{.Name}}{{ print "\n" }}{{end}}
```

And sourcing this bash completion:
```sh
#/usr/bin/env bash
_robo_completion() {
  if [ "${#COMP_WORDS[@]}" != "2" ]; then
    return
  fi
  COMPREPLY=($(compgen -W "$(robo)" -- "${COMP_WORDS[1]}"))
}

complete -F _robo_completion robo
```

Works nicely. But it's annoying adding the list template in each `robo.yml` file.

**Comment by yields** [comment]
Nice use case, perhaps we should use `github.com/alecthomas/colour`, from it's `TTY()` docs:

```bash
$ go doc colour.TTY
func TTY(w io.Writer) Printer
    TTY creates a Printer that colourises output if w is a terminal, or strips
    formatting if it is not.
```

Other solutions might be to add completions automatically for shells like zsh and bash, or a `--json` flag that will output JSON?

I'll see if I have 30m this weekend to whip up a PR.

**Comment by yields** [comment]
Actually haven't thought about it a bit, it seems like `-q` is the simplest way to go about it, similar to how docker outputs verbose info but only lists the IDs with `-q`.

**Comment by rafi** [comment]
Exactly my thoughts @yields, same as docker lists only the IDs with `-q`.

**Comment by yields** [comment]
Released [v0.7.0](https://github.com/tj/robo/releases/tag/v0.7.0):

```bash
  λ ~ robo --version
0.7.0
  λ ~ robo -q
dev
stage
prod
...
```
